From f5810c7ab8542e8ac471bfeea9a83f47898d5fdf Mon Sep 17 00:00:00 2001
From: Jaya Praveen Gangavarapu <jaya.praveenx.gangavarapu@intel.com>
Date: Thu, 8 Oct 2015 12:48:52 +0530
Subject: [PATCH] bt,lnp: Config name changed to CONFIG_INTEL_LD.

Changed the config name from CONFIG_ST_LD to CONFIG_INTEL_LD.

Change-Id: Iab34c666aad783f3246a81f05a9e077df3d5fc0c
Signed-off-by: Reddy, Praveen <praveen.reddy@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-2119
---
 arch/x86/configs/x86_64_debug_defconfig | 2 +-
 arch/x86/configs/x86_64_defconfig       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/x86/configs/x86_64_debug_defconfig b/arch/x86/configs/x86_64_debug_defconfig
index 6186d42..254e481 100644
--- a/arch/x86/configs/x86_64_debug_defconfig
+++ b/arch/x86/configs/x86_64_debug_defconfig
@@ -1211,7 +1211,7 @@ CONFIG_EEPROM_93CX6=m
 # Texas Instruments shared transport line discipline
 #
 # CONFIG_TI_ST is not set
-CONFIG_INTEL_ST_LD=m
+CONFIG_INTEL_LD=m
 # CONFIG_SENSORS_LIS3_I2C is not set
 
 #
diff --git a/arch/x86/configs/x86_64_defconfig b/arch/x86/configs/x86_64_defconfig
index e1a27a37..c5877ee 100644
--- a/arch/x86/configs/x86_64_defconfig
+++ b/arch/x86/configs/x86_64_defconfig
@@ -1208,7 +1208,7 @@ CONFIG_EEPROM_93CX6=m
 # Texas Instruments shared transport line discipline
 #
 # CONFIG_TI_ST is not set
-CONFIG_INTEL_ST_LD=m
+CONFIG_INTEL_LD=m
 # CONFIG_SENSORS_LIS3_I2C is not set
 
 #
-- 
1.9.1

